requirements-dev.txt

```diff
@@ -4,7 +4,7 @@ bluesky
 codecov
 coverage
 flake8
-pytest >=3.9
+pytest >=5.1.0
```

Comment:
Was this necessary, or just an attempt to get the CI working due to the `attrs` issue? If we don't actually need any pytest 5.1 features, let's revert this to 3.9.

Reply:
This was attempt to for CI issue.